app/constants/colorConstants.ts

```diff
@@ -0,0 +1 @@
+const colors = {};
```

Comment:
remove this file, we will be adding it to tailwind config

Reply:
okay